docs/DEV-3.-Frontend.md

```diff
@@ -1 +1,111 @@
 # Frontend Deployment for Development
+
+
+## The Microfrontend configuration
+The Field Mapping Tasking Manager is a web application that helps users coordinate mapping efforts in OpenStreetMap. It is built using a microfrontend architecture, which means that the application is composed of multiple small frontend modules that can be developed, tested, and deployed independently.
+
+FMTM uses Webpack remote modules to dynamically load code from other microfrontend applications, which allows developers to develop, test, and deploy smaller and more manageable parts of code independently. This improves the performance and scalability of the application.
+
+
+
+## Setting up the development environment
+### 1. Get the project code
+  Clone the Field Mapping Tasking Manager repository from GitHub using the following command:
+
+`git clone https://github.com/hotosm/tasking-manager.git`
+
+### 2. Install dependencies
+  Navigate to the react frontend directory: `cd src/frontend/main` 
+  
+  Install the dependencies by running the following command:`npm install`
+
+### 3. Run the project
+Using the following command, run the project:
```

Comment:
Both parts of the microfrontend are required for it to work.
Please change this section slightly to explain:
- It is possible to run each microfrontend separately in a separate terminal (with a npm install & npm run start:live), but the easiest option would be to use docker.